simulator/simulator.go

```diff
@@ -0,0 +1,260 @@
+package simulator
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/log"
+)
+
+// Manager provides methods for issuing Simulator commands
+type Manager interface {
```

Comment:
Is a better name for this `iOSSimulatorManager`

Reply:
disagree, it will be used as _simulator.Manager_. no need to add the package name in the name.

Reply:
Ah good point.